### ISSUE TITLE
Tagging select option search may be confusing

### ISSUE THREAD

**Issue by lisettemeij** [landportal/drupal#229]
I'm experiencing weird issues when selecting a country or a concept in the new tagging system. When starting to typ, the fields give you suggestions from the taxonomies but also apparently keywords that don't actually exist in the taxonomy - giving you an error.

Example country taxonomy:
Sometimes it selects a country without a capital letter, for example: I type in "Brazil", it selects "brazil", and then when I want to publish it - it says "illegal choice detected" and the geographical focus field appears empty. Then I type in Brazil again and I have the choice to select "brazil" and "Brazil". The last one is the only one that exists in the country taxonomy, so that one does not give an error. However, even when I start typing "Brazil", it will give "brazil" on the top of the list so if I hit enter quickly, that's the one it selects -- giving me the error. See below:

<img width="1280" alt="screen shot 2017-01-25 at 12 36 48" src="https://cloud.githubusercontent.com/assets/6312043/22289418/b7908dea-e2fb-11e6-9d88-1941d4c5bd75.png">

(Unfortunately not in the screenshot, but the geographical focus field was empty below).

Second example LandVoc concepts:
I typed in "security of tenure" and then I'll hit "security of tenure (land)", which is the LandVoc concept that I know. The field gives "security of tenure" (as is) as a suggestion and I'm able to select it and everything. Though I know that does not exist in LandVoc, nor is it listed as a synonym of "security of tenure (land)" -> https://landportal.info/voc/landvoc/concept/security-tenure-land
I had more concepts in the fields so it didn't give me an error, but when I was redirected back because of the geographical focus error, I noticed that the concepts that didn't exist in LandVoc were automatically deleted from the field.

So to sum up: the suggestions given by the fields should ONLY be those concepts that actually exist in our taxonomies !

**Comment by jcr** [comment]
From hangouts
> ok I think I can explain it at least
> Lisette what you actually "validated" / put in the field is what you typed
> not the selection, which is confusing because in the dropdown, apparently, the first item *is* what you type not an existing term
(...)
> what is true though is that is should validate the options and not let you put as a tag (letting you think its ok) something that doesn't exists
> maybe just validation can help

**Comment by jcr** [comment]
fixed: no more "new" tags
